src/test/unit/calculateCoordinates.test.ts

```diff
@@ -0,0 +1,91 @@
+import * as chai from 'chai';
+import chaiShallowDeepEqual from 'chai-shallow-deep-equal';
+import { getCoordinates } from '../../calculateCoordinates';
+import conf from '../../conf';
+import { ERROR_MESSAGES } from '../../conf/constants';
+import DATA from '../utilities/testData';
+chai.use(chaiShallowDeepEqual);
+
+const expect = chai.expect;
+
+describe('Unit tests for getCoordinates', () => {
```

Comment:
It looks like this test file is not mocking the Geocode API. Would we want to do that on a real application?

Reply:
Yes right, `getCoordinates` method is not using `Geocode API` directly in the code, but `geocode` method can be stubbed using Sinon spy.

Reply:
I am not using Geocode API  directly in `getCoordinates.ts`, using `node-geocoder` module. But `gecode` method can be stubbed using Sinon spy.